inst/sympref.m

```diff
@@ -420,6 +420,13 @@
 %! delete('tmp_python_cmd.py')
 
 %!test
+%! warning ('off', 'OctSymPy:sympref:invalidarg', 'local');
```

Comment:
Ah, ok. I missed that you had a test runner for Matlab, that's a good thing.

We could make it more like Octave, with a separate function scope for each distinct test "paragraph", if you're open to that.